crates/proof/executor/TEST_FIXTURES.md

```diff
@@ -0,0 +1,33 @@
+# `kona-executor` test fixtures
+
+The `StatelessL2Builder` type uses static test data fixtures, located in [`testdata`](./testdata/). The test data
+fixtures include:
+* The `RollupConfig` of the chain that the block belongs to.
+* The parent block header, which we apply state on top of.
+* The payload attributes for building the new block.
+* A `rocksdb` database containing the witness data for stateless execution of the block building job.
+
+Sometimes, updates in the block building code can add new state accesses, requiring these fixtures to be re-generated.
+Head over to [`src/builder/core.rs`](./src/builder/core.rs) and navigate to the test module at the bottom of the file.
+
+A commented out test named `create_fixture` is included at the top of the module:
+
+```rs,no_run
+#[tokio::test(flavor = "multi_thread")]
+async fn create_fixture() {
+    let fixture_creator = crate::test_utils::ExecutorTestFixtureCreator::new(
+        "<rpc_url>",
+        <block_number>,
+        PathBuf::from(env!("CARGO_MANIFEST_DIR")).join("testdata"),
+    );
+    fixture_creator.create_static_fixture().await;
+}
+```
```

Comment:
One suggestion is we could hoist this into an example. Kona already has a few [workspace-level examples](https://github.com/op-rs/kona/tree/main/examples) that can be run with `cargo run --example <example>`. This way we don't have commented out code

Reply:
this sounds better - will do in the am